lib/utils.js

```diff
@@ -630,9 +630,23 @@ function _populateObj(obj) {
  */
 
 exports.getValue = function(path, obj, map) {
-  return mpath.get(path, obj, '_doc', map);
+  return mpath.get(path, obj, getValueLookup, map);
 };
 
+/*!
+ * ignore
+ */
+
+const mapGetterOptions = Object.freeze({ getters: false });
+
+function getValueLookup(obj, part) {
+  const _from = obj && obj._doc ? obj._doc : obj;
```

Comment:
Nitpick:
```suggestion
  const _from = obj?._doc || obj;
```

Reply:
Works for me, we do have one place where we use `?.` in runtime code and no complaints so far. So this should work nicely.